### ISSUE TITLE
my Colab PRO+ Cannot run my code more than 12 hours

### ISSUE THREAD

**Issue by karlisetiawan** [googlecolab/colabtools#2878]
**Describe the current behavior**
my Colab PRO+ cannot run my code more than 12 hours. i do not know why.

**Describe the expected behavior**
Colab PRO+ should run my for 24 hours as the products description written.

**What web browser you are using**
Firefox 101.0.1 (64-bit)

**Comment by corrieann** [comment]
Pro+ users have the additional ability to enable background execution for a limited number of notebooks which will continue execution after you close your browser to a maximum runtime limit of up to 24 hours. You can enable background execution in Notebook settings. Background execution only applies during an execution state: idle times will result in VM deletion. Your output will be saved to Drive after the completion of each cell execution, even if your browser is closed. Resources are not guaranteed and usage limits still apply.

TLDR: Make sure to enable background execution under Edit -> Notebook settings and note that if your VM is idle, it may still be terminated.

If you're still running into issues, please file feedback using Help -> Send feedback
